app/hooks/useSettingsAction.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
```suggestion
import * as React from "react";
```

Prefer for consistency